organize/models.py

```diff
@@ -92,6 +93,20 @@ def create_event(self):
 
         return event
 
+    def has_past_team_members(self):
+        """ For repeated events, check whether there are any common
+        team members who applied to organize again
+        """
+        previous_events = Event.objects.filter(city=self.city,
+                                               country=self.get_country_display())
```

Comment:
I believe the value stored in the Events database is `self.country`, not `self.get_country_display()`